lib/components/base/ConditionalNuxtLink.vue

```diff
@@ -7,15 +7,6 @@
   </span>
 </template>
 
-<script setup>
-defineProps({
-  to: {
-    type: String,
-    required: true,
-  },
-  isLink: {
-    type: Boolean,
-    required: true,
-  },
-})
+<script setup lang="ts">
+defineProps<{ to: string; isLink: false }>()
```

Comment:
```suggestion
defineProps<{ to: string; isLink: boolean }>()
```

Reply:
tbh, this shouldn't even be valid code...

thanks for the check